robot/packages/cloud_package_manager.go

```diff
@@ -181,17 +133,22 @@ func (m *cloudManager) Sync(ctx context.Context, packages []config.PackageConfig
 		if ok {
 			if existing.thePackage.Package == p.Package && existing.thePackage.Version == p.Version {
 				m.logger.Debug("  Package already managed, skipping")
-
 				newManagedPackages[PackageName(p.Name)] = existing
-				delete(m.managedPackages, PackageName(p.Name))
 				continue
 			}
 			// anything left over in the m.managedPackages will be cleaned up later.
 		}
 
 		// Lookup the packages http url
 		includeURL := true
-		resp, err := m.client.GetPackage(ctx, &pb.GetPackageRequest{Id: p.Package, Version: p.Version, IncludeUrl: &includeURL})
+		_ = fmt.Sprintf("%s/%s", runtime.GOOS, runtime.GOARCH)
+		resp, err := m.client.GetPackage(ctx, &pb.GetPackageRequest{
+			Id:      p.Package,
+			Version: p.Version,
+			// Type:       PackageTypeToProto(p.Type),
+			// Platform:   &platform,
```

Comment:
Pre-merge: need to only platform for modules, not for ML models. Otherwise it can't find them.